dataprofiler/labelers/data_labelers.py

```diff
@@ -33,6 +33,8 @@ def train_structured_labeler(data, save_dirpath=None, epochs=2):
             "Input data must be either a `pd.DataFrame` or a "
             "`data_profiler.Data` and not of type `TextData`."
         )
+    if default_label and not isinstance(default_label, str):
+        raise TypeError("Default label must be a string.")
```

Comment:
is this necessary? wont' the validate from the set_params already raise this error?